ExtCore/Pervasive.fs

```diff
@@ -806,7 +791,7 @@ module Lazy =
 #endif
 
 /// Additional functional operators on options.
-[<RequireQualifiedAccess; CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
+[<RequireQualifiedAccess>]
```

Comment:
Why was the ``CompilationRepresentation`` attribute removed here?

Reply:
I think it was due to a thing @vasily-kirichenko said, that it's no longer needed.